.github/workflows/code-health.yml

```diff
@@ -16,22 +16,34 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
curious: why is the PR called "Test Double infrastructure"?

Reply:
so is my understanding correct that in this PR we are introducing mockery to auto-generate mocks?

Reply:
i can use a better name. we're introducing auto-generated mocks (including workflow action) + an example of how to use it

Reply:
> so is my understanding correct that in this PR we are introducing mockery to auto-generate mocks?

My understanding was that we were already autogenerating the mock similar to what the CLI does. 🤔 

> any suggestion for a better name?

Maybe instead of changing the name you could add additional clarifications int he description of this PR

Reply:
no, we were creating the mocks ourshelves.  thanks, I'll add more details to the PR description

Reply:
clarification added to PR description, and title changed, please let me know if you would add something else

Reply:
@marcosuma if the question was about the word "Double", it's because I think that word is more precise.   Mocks, stubs, spies,  etc. are types of test doubles but I understand that nowadays the word "mock" is used as a synonym for test double so I've changed the PR title.

Reply:
thank you! #til - very rarely I've heard of test doubles. All good on my side